.cirrus.yml

```diff
@@ -70,7 +70,7 @@ task:
   env:
     HOME: /root
     CIRRUS_WORKING_DIR: /home/runc
-    GO_VERSION: "1.19"
+    GO_VERSION: "1.20.2"
```

Comment:
Given that this PR has a go1.19 variant, perhaps it would be good to move the go1.20 update separate. 

That way, this PR will be testing the go1.19 branch (which would also be the case if we backport this to 1.1.x (?)). We already had a run of CI with 1.20 on this PR, and once the 1.20 update (separate PR) is merged, `main` will be testing that again.

Reply:
Sure. Done.